certified-connectors/LassoX/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4001 @@
+                      "title": "Value"
+                    },
+                    "possibleError": {
+                      "type": "boolean",
+                      "description": "Whether there is there a potential error in the data",
+                      "title": "Possible Error"
+                    },
+                    "error": {
+                      "type": "boolean",
+                      "description": "Whether there is an error in the data",
+                      "title": "Error"
+                    }
+                  },
+                  "description": "(OBJECT) Hensatte Forpligtelser",
+                  "title": "Provisions"
+                },
+                "reportingCompanyName": {
+                  "type": "object",
+                  "properties": {
+                    "unit": {
+                      "type": "string",
+                      "description": "Unit of the value, e.g. DKK or Percentage",
+                      "title": "Unit"
+                    },
+                    "value": {
+                      "type": "string",
+                      "description": "The value of the key figure",
+                      "title": "Value"
+                    },
+                    "possibleError": {
+                      "type": "boolean",
+                      "description": "Whether there is there a potential error in the data",
+                      "title": "Possible Error"
+                    },
+                    "error": {
+                      "type": "boolean",
+                      "description": "Whether there is an error in the data",
+                      "title": "Error"
+                    }
+                  },
+                  "description": "(OBJECT) Virksomhedens Navn",
+                  "title": "Reporting Company Name"
+                },
+                "result": {
+                  "type": "object",
+                  "properties": {
+                    "unit": {
+                      "type": "string",
+                      "description": "Unit of the value, e.g. DKK or Percentage",
+                      "title": "Unit"
+                    },
+                    "value": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "The value of the key figure",
+                      "title": "Value"
+                    },
+                    "possibleError": {
+                      "type": "boolean",
+                      "description": "Whether there is there a potential error in the data",
+                      "title": "Possible Error"
+                    },
+                    "error": {
+                      "type": "boolean",
+                      "description": "Whether there is an error in the data",
+                      "title": "Error"
+                    }
+                  },
+                  "description": "(OBJECT) \u00c5rets Resultat",
```

Comment:
@emilbm This value has non-English characters which is not allowed and will not render properly in the Power Automate. Please replace all the non-English characters with English characters.